documentation/How-to-add-a-new-corpus-to-Ianalyzer.md

```diff
@@ -103,3 +115,13 @@ Optional flags:
 - `-d` specifies that an existing index of the same name should be deleted first (if not specified, defaults to false, meaning that extra data can be added while existing data is not overwritten)
 
 The start and end date flags are passed on the `sources` function of the corpus (see above). If you did not utilise them there, they will not do anything.
+
+## Validation
+
+The `CorpusDefinition` class has no built-in validation. However, once you start using the corpus, many of the properties defined in the python class will be loaded into the `CorpusConfiguration` database model. This step does include some validation, so it may raise errors. You ran run the import script with `yarn django loadcorpora`. It is also run when you start a development server with `yarn start-back`.
```

Comment:
```suggestion
The `CorpusDefinition` class has no built-in validation. However, once you start using the corpus, many of the properties defined in the python class will be loaded into the `CorpusConfiguration` database model. This step does include some validation, so it may raise errors. You can run the import script with `yarn django loadcorpora`. It is also run when you start a development server with `yarn start-back`.
```